### PR TITLE
Tarog star update graphql actions

### PR DESCRIPTION
**Pull request by TarogStar**
To reduce transactions, those using the GraphQL endpoint can now add Play Count to boost their rewards.  The new Play Count variable is not required and defaults to 1.  Also added an action to Charge AP using GraphQL, which should close issue #968

### CODE REVIEW

**Review by longfin — APPROVED** [review]
LGTM! 

@planetarium/9c-backend Could you review it?

File: NineChronicles.Headless/GraphTypes/ActionMutation.cs [hunk 1]
```diff
@@ -157,6 +164,7 @@ public ActionMutation(NineChroniclesNodeService service)
                             costumes = costumeIds,
                             equipments = equipmentIds,
                             foods = consumableIds,
+                            playCount = playCount < 1 ? 1 : playCount
```
Comment (moreal) [inline NineChronicles.Headless/GraphTypes/ActionMutation.cs:167]: In my thought, it'll be better to throw errors when unexpected negative numbers came than forcing to 1 implicitly.

**Review by area363 — APPROVED** [review]
👍

**Review by moreal — APPROVED** [review]
🚀

File: NineChronicles.Headless/GraphTypes/ActionMutation.cs (hunk 1 above)
Comment (TarogStar) [reply NineChronicles.Headless/GraphTypes/ActionMutation.cs]: I was wondering whether to include that myself.  I adjusted the branch to remove that extra code.

**Review by longfin — APPROVED** [review]

**Pull request merged**